frontend/src/components/LineView.js

```diff
@@ -122,23 +122,27 @@ const LineView = ( {
           </div>
 
           <h1>
-
             {gurmukhi
               ? gurmukhi
                 .split( ' ' )
                 .map( ( word, index ) => (
-                  <a
-                    // eslint-disable-next-line react/no-array-index-key
-                    key={index}
-                    href={getDictionaryLink( stripVishraams( toUnicode( word ) ) )}
-                    target="_blank"
-                    rel="noreferrer"
-                  >
-                    {word}
-                  </a>
+                  <div>
+                    <a
+                      // eslint-disable-next-line react/no-array-index-key
+                      key={index}
+                      className="gurmukhi"
+                      href={getDictionaryLink( stripVishraams( toUnicode( word ) ) )}
+                      target="_blank"
+                      rel="noreferrer"
+                    >
+                      {word}
+                    </a>
+                    <span>{toSyllabicSymbols( word )}</span>
+                  </div>
                 ) )
                 .reduce( ( prev, curr ) => [ prev, ' ', curr ] )
               : <Loader size="1em" />}
+            {gurmukhi && ( <span className="syllable-count">{countSyllables( gurmukhi )}</span> )}
```

Comment:
How have you lined up the syllables, since they don't appear to be grouped to a word?

Reply:
The final count is not grouped to the word, but yes it's lined up as the image shows in the rounded square icon

Reply:
I see, I thought it was somehow the individual word counts, my bad. Please add a newline above it just to visually separate it from the "words" code block above

Reply:
```suggestion

            {gurmukhi && ( <span className="syllable-count">{countSyllables( gurmukhi )}</span> )}
```